src/pages/CatEdit.js

```diff
@@ -1,7 +1,8 @@
 import React, { Component } from 'react';
 import { Form, FormGroup, Label, Input, Button } from 'reactstrap';
 // react-router import
-import { Redirect } from 'react-router-dom'
+import { Redirect } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
```

Comment:
SInce these are both coming from React router, you can import them all together. `import { Redirect, NavLink } from 'react-router-dom'`